packages/core/.eslintrc.js

```diff
@@ -8,7 +8,6 @@ module.exports = {
     createDefaultProgram: true,
   },
   rules: {
-    "@nomiclabs/hardhat-internal-rules/only-hardhat-plugin-error": "error",
```

Comment:
I understand why we need to remove hh as a dependency and agree, but it's unfortunate that we're losing this linting to enforce ignition errors. I would love to find a solution that would solve both problems, but aside from writing our own eslint plugin to accomplish the same thing, I'm not sure what that solution would actually be.

Reply:
Maybe parametrize the rule? `only-custom-error` or something like that?